src/OVAL/probes/unix/linux/rpmverify_probe.c

```diff
@@ -303,19 +303,14 @@ typedef struct {
 } rpmverify_bhmap_t;
 
 const rpmverify_bhmap_t rpmverify_bhmap[] = {
-        { "nodeps",        (uint64_t)VERIFY_DEPS      },
```

Comment:
Is there any way how to keep those items with rpmlib >= 4.15? These are defined by the [OVAL spefification](https://oval.mitre.org/language/version5.11/ovaldefinition/documentation/linux-definitions-schema.html#RpmVerifyBehaviors). They are marked as deprecated, but we should still be able to process them.

Reply:
Hi @ignatenkobrain , do you have any information about this?

Reply:
VERIFY_DEPS was never a part of flags for rpmVerifyFile. This is a flag for rpmcliVerify, which is not used in this probe at all. Whatever this probe was supposed to do regarding the --nodeps flag it never did.